modules/otel-troubleshoot-logging-exporter-stdout.adoc

```diff
@@ -4,9 +4,9 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="debug-exporter-to-stdout_{context}"]
-= Debug exporter
```

Comment:
Question: should this file be renamed to `otel-troubleshoot-debug-exporter-stdout.adoc`?

Reply:
Thank you. Fixed.